pallets/pallet-deposit-storage/src/deposit.rs

```diff
@@ -59,10 +59,10 @@ pub enum FixedDepositCollectorViaDepositsPalletError {
 impl From<FixedDepositCollectorViaDepositsPalletError> for u16 {
 	fn from(value: FixedDepositCollectorViaDepositsPalletError) -> Self {
 		match value {
-			FixedDepositCollectorViaDepositsPalletError::DepositAlreadyTaken => 0,
-			FixedDepositCollectorViaDepositsPalletError::DepositNotFound => 1,
-			FixedDepositCollectorViaDepositsPalletError::FailedToHold => 2,
-			FixedDepositCollectorViaDepositsPalletError::FailedToRelease => 3,
+			FixedDepositCollectorViaDepositsPalletError::DepositAlreadyTaken => 1,
```

Comment:
Yes because each case is separated from the previous one by a `u8::MAX` value. So it would not possible to tell the difference between a `u8::MAX` of one case and a `u8::MAX + 0` of another case.

Reply:
I see.